bigchaindb/core.py

```diff
@@ -232,12 +232,13 @@ def commit(self):
         # register a new block only when new transactions are received
         if self.block_txn_ids:
             self.bigchaindb.store_bulk_transactions(self.block_transactions)
-            block = Block(app_hash=self.block_txn_hash,
-                          height=self.new_height,
-                          transactions=self.block_txn_ids)
-            # NOTE: storing the block should be the last operation during commit
-            # this effects crash recovery. Refer BEP#8 for details
-            self.bigchaindb.store_block(block._asdict())
+
```

Comment:
nit: There is not change here. pep8? 

I don't understand the fix, maybe Work in progress?

Reply:
Ah now I see, we are storing empty blocks, completely overlooked the indentation.